lib/inferno/jobs/invoke_validator_session.rb

```diff
@@ -6,13 +6,17 @@ class InvokeValidatorSession
       def perform(suite_id, validator_name, validator_index)
         suite = Inferno::Repositories::TestSuites.new.find suite_id
         validator = suite.fhir_validators[validator_name.to_sym][validator_index]
-
         response_body = validator.validate(FHIR::Patient.new, 'http://hl7.org/fhir/StructureDefinition/Patient')
-
         if response_body.start_with? '{'
           res = JSON.parse(response_body)
           session_id = res['sessionId']
-          # TODO: (FI-2311) store this session ID so it can be referenced as needed
+          session_repo = Inferno::Repositories::ValidatorSessions.new
+          begin
+            session_repo.create(test_suite_id: suite_id, validator_session_id: session_id, \
+                                validator_name:)
+          rescue Sequel::ValidationFailed => e
+            puts e.message
```

Comment:
Thanks for pointing this out. I think we should not catch and log this exception because it is not actually ignore-able. If we cannot save a validated session, it defeats the purpose.